README.md

```diff
@@ -1,142 +1,126 @@
+| ARTIFACT_IMAGE_SUFFIX |          | The suffix of the artifact ImageStream, default is `artifact`         |
+| GIT_URL               | Yes      | URL to git repository, ex: `https://github.com/sklintyg/frontend.git` |
+| GIT_CI_BRANCH         | Yes      | Branch in git repository, ex: `master`                                |
+
+To create a frontend-pipeline using the template, you make sure to first login to OpenShift and then run the following command.
 
-To create a frontend-pipeline using the template, you make sure to first login to OpenShift and then run the following command. 
 ```
 `oc process -f pipelinetemplate-build-frontend.yaml -p APP_NAME=webcert-frontend -p RELEASE_VERSION=2021-2 -p GIT_URL=https://github.com/sklintyg/frontend.git -p GIT_CI_BRANCH=master | oc apply  -f -`
 ```
 
 ## Running storybook
 
-Storybook can be used to develop and test components within the common package. Storybook will hot-reload changes in common. 
+Storybook can be used to develop and test components within the common package. Storybook will hot-reload changes in common.
 
 - Start storybook: `yarn workspace @frontend/common storybook`
 
 ## Running tests
 
-Jest is used for executing tests. 
+Jest is used for executing tests.
 
-To run tests in Webcert:  `yarn workspace @frontend/webcert test`
+To run tests in Webcert: `yarn workspace @frontend/webcert test`
 
-To run tests in Common:  `yarn workspace @frontend/common test` (add `--watchAll` if you want the test runner to run the tests for any change you make.)
+To run tests in Common: `yarn workspace @frontend/common test` (add `--watchAll` if you want the test runner to run the tests for any change you make.)
 
 See [Jest CLI Options](https://jestjs.io/docs/en/cli) for more info what you can do when executing tests.
 
 ## Writing tests
 
-Below are recommended type of tests for a component. We are looking into [jest-axe](https://www.npmjs.com/package/jest-axe) for accessibility testing.
+Below are recommended types of tests for a component. We are looking into [jest-axe](https://www.npmjs.com/package/jest-axe) for accessibility testing.
 
 Smoke test that checks if the component can be rendered without crashing. Ex:
 
 ```javascript
 it('renders without crashing', () => {
-   const question = createQuestionWithTextValue()
-   const div = document.createElement('div')
-   ReactDOM.render(<UvText question={question} />, div)
- })
+  const question = createQuestionWithTextValue()
+  const div = document.createElement('div')
+  ReactDOM.render(<UvText question={question} />, div)
+})
 ```
+
 Tests that verifies the components behavior from a user perspective. Use React Testing Library.
+
 ```javascript
 it('displaying empty value', () => {
   const question = createQuestion({ type: CertificateDataValueType.TEXT })
   const { getByText } = render(<UvText question={question} />)
   getByText(/Ej angivet/i)
 })
 ```
+
 Test with snapshot to verify that changes in the component was intentional.
```

Comment:
We could remove snapshot-testing all together. We did try it out but doesn't find valuable enough.